mqtt5/README.md

```diff
@@ -6,16 +6,34 @@ This document defines how to describe MQTT 5-specific information on AsyncAPI.
 
 ## Version
 
-Current version is `0.1.0`.
+Current version is `0.2.0`.
 
 
 <a name="server"></a>
 
 ## Server Binding Object
 
-This object MUST NOT contain any properties. Its name is reserved for future use.
+This object contains information about the server representation in MQTT5.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="serverBindingObjectSessionExpiryInterval"></a>`sessionExpiryInterval` | [Schema Object][schemaObject] \| integer | Session Expiry Interval in seconds.
```

Comment:
> Session Expiry Interval in seconds.

@sudoshreyansh if the value is a Schema Object, then the description above is not true. Please adjust.

Reply:
@fmvilas I have updated the description.